os-appium-tests/tests/utils/locators/LocatorsInAppBrowser.ts

```diff
@@ -0,0 +1,31 @@
+export const LOCATORS = {
+    HTTPS_VALID_URL: 'button_https_valid_url',
+    HTTP_VALID_URL: 'button_http_valid_url',
+    CLOSE_BUTTON: 'Input_Closebuttoncaption'
+};
+
+const SELECTORS = {
+    ANDROID: {
+        BEM_VINDO_MENSAGEM_FINANCAS: '//*[contains(@text, "Bem-vindo ao Portal das")]',
+        MENSAGEM_EUNOPS_PAGE: '//*[contains(@text, "eunops")]',
+    },
+    IOS: {
+        BEM_VINDO_MENSAGEM_FINANCAS: '//XCUIElementTypeStaticText[@label="Bem-vindo ao Portal das Finanças"]',
```

Comment:
Why are we using Portal das Finanças in this test? We should create our own web app to avoid having tests crashing because the layout of the page was changed

Reply:
During all the sprint we have done an analysis to realize what could be the best approach. So, it would be better an own web app or an external page that , in our opinion, could be reliable? We reach a commitment that used pages could be the best approach avoiding exposing  our own created web pages.

Reply:
And we have a dependency, use public endpoints during tests so the tests can run in SauceLabs, AWS tools.

Reply:
I understand your problems, but I do not agree with the fundaments. We do not own the page being used to our tests, and therefore, the tests can break without prior notice. We have public servers (intranet.outsystems.net, for example) that can be used to host a public page (we already have MABS there). For me, relying on a public webpage is a dangerous approach.

Reply:
Solved with URL:

https://enmobile11.outsystemsenterprise.com/VFO_DealWithIt/